src/DssSpell.t.sol

```diff
@@ -225,14 +197,13 @@ contract DssSpellTest is DSTest, DSMath {
         // Test for spell-specific parameters
         //
         spellValues = SpellValues({
-            deployed_spell:                 address(0x961eEE551d0Fe5FF8B88db5441875564ac967f09),        // populate with deployed spell if deployed
-            deployed_spell_created:         1619196280,        // use get-created-timestamp.sh if deployed
-            previous_spell:                 address(0xDb0D1af4531F59E4E7EfA4ec0AcADec7518D42B6),        // supply if there is a need to test prior to its cast() function being called on-chain.
-            previous_spell_execution_time:  1619372356,        // Time to warp to in order to allow the previous spell to be cast ignored if PREV_SPELL is SpellLike(address(0)).
-            office_hours_enabled:           true,              // true if officehours is expected to be enabled in the spell
-            expiration_threshold:           weekly_expiration  // (weekly_expiration,monthly_expiration) if weekly or monthly spell
+            deployed_spell:                 address(0),        // populate with deployed spell if deployed
+            deployed_spell_created:         1616432122,                 // use get-created-timestamp.sh if deployed
+            previous_spell:                 address(0),        // supply if there is a need to test prior to its cast() function being called on-chain.
+            previous_spell_execution_time:  1616599434,                 // Time to warp to in order to allow the previous spell to be cast ignored if PREV_SPELL is SpellLike(address(0)).
```

Comment:
I would suggest to add the prev spell `address` and `timestamp` as this have not yet been executed:
https://vote.makerdao.com/executive/system-surplus-buffer-increase-and-burn-parameter-changes-yfi-a-addition-to-liquidations-2-0?network=mainnet#proposal-detail